src/main/java/org/carlmontrobotics/robotcode2023/subsystems/Roller.java

```diff
@@ -39,14 +46,11 @@ public boolean runsWhenDisabled() {
         };
     };
 
-    public Roller() {
+    public Roller(Drivetrain dt) {
         led.setLength(ledBuffer.getLength());
         setLedColor(defaultColor);
-        //SmartDashboard.putData(this);
-
-        // TODO: Get proper values for the speeds and timings. For future pull requests,
-        // do not merge if this is not deleted or speeds/timings have not been determined
-        
+        // SmartDashboard.putData(this);
+        this.dt = dt;
```

Comment:
See other comment. Roller should not access drivetrain directly. The command to drive to the nearest goal should.